src/pages/Login.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { Link } from 'react-router-dom';
+import { FaAngleRight } from "react-icons/fa"
+
+export const Login = () => {
+    return (
+        <main>
+            <section className="login">
+                <form action="" method="get" className="form p-m m-m br-s">
+                    <h1 className="m-s p-s txt-center">Login</h1>
+
+                    <div className="flex-column">
+                        <label htmlFor="Email" className="m-xs">Email Address:</label>
+                        <input type="email" name="Email" className="input-box p-s m-xs br-s" placeholder="abc@xyz.com" />
+                    </div>
+
+                    <div className="flex-column">
+                        <label htmlFor="Password" className="m-xs">Password:</label>
+                        <input type="password" name="Password" className="input-box p-s m-xs br-s" placeholder="******" />
+                    </div>
+
+                    <div className="flex-row flex-space-between">
+                        <div className="checkbox-div">
+                            <input type="checkbox" name="Remember-me" className="br-xs p-xs m-xs txt-m" />
```

Comment:
Same as the suggestion mentioned in Signup section

Reply:
done